src/coreclr/jit/llvm.cpp

```diff
@@ -1424,6 +1444,119 @@ void Llvm::importStoreInd(GenTreeStoreInd* storeIndOp)
     }
 }
 
+// Copies endOffset - startOffset bytes, endOffset is exclusive.
+unsigned Llvm::buildMemCpy(Value* baseAddress, unsigned startOffset, unsigned endOffset, Value* srcAddress)
+{
+    Value* paddingDestAddress = _builder.CreateGEP(baseAddress, _builder.getInt32(startOffset));
+
+    const unsigned paddingSize = endOffset - startOffset;
+    // copy the bytes in the padding
+    _builder.CreateMemCpy(paddingDestAddress, llvm::Align(), srcAddress, llvm::Align(), paddingSize);
+
+    return paddingSize;
+}
```

Comment:
```suggestion
// Copies endOffset - startOffset bytes, endOffset is exclusive.
unsigned Llvm::buildMemCpy(Value* baseAddress, unsigned startOffset, unsigned endOffset, Value* srcAddress)
{
    Value* destAddress = _builder.CreateGEP(baseAddress, _builder.getInt32(startOffset));
    unsigned size = endOffset - startOffset;

    _builder.CreateMemCpy(destAddress, llvm::Align(), srcAddress, llvm::Align(), size);

    return size;
}
```
Doesn't seem like there is great a reason to make this padding-copying-specific.